### DISCUSSION TITLE
A question about moving files.

### DISCUSSION THREAD

**Discussion by audiogamer22** [nianeyna/ao3downloader#100] [Q&A] [answered]
If I've moved files out of the downloads folder, then move them bakc and paste the same URL as before, will the program search through all those files and not download them again?

At the moment we can't choose where to download our files to, which is fine, but as someone who downloads multiple genres? Of fan fic, I'd find it useful to have something in place in case I've not got all the files from the download.

**Comment by nianeyna** [comment, marked as answer]
yes, the program will check the existing files in the "downloads" folder
against records in the "logs" folder to decide what to skip when the
download restarts.
